db/data/20210518183334_create_notifications_for_communications.rb

```diff
@@ -0,0 +1,9 @@
+class CreateNotificationsForCommunications < ActiveRecord::Migration[5.2]
+  def up
+    BackfillNotifications.call
+  end
+
+  def down
+    raise ActiveRecord::IrreversibleMigration
```

Comment:
I don't think it provides any real value, but we could just destroy all notifications as a means of reversing this. But this is probably the better choice.

Reply:
This is just the default that the data migrate tool gives us, because making them reversible is usually not worth the trouble